addons/website/static/src/js/menu/edit.js

```diff
@@ -63,18 +63,28 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         url.searchParams.delete('enable_editor');
         url.searchParams.delete('with_loader');
         window.history.replaceState({}, null, url);
+
+        if (this._editorAutoStart) {
+            $('body').block({overlayCSS: {
+                backgroundColor: '#000',
+                opacity: 0.5,
```

Comment:
The 0.5 opacity seems weird visually, it is too much.
I would just keep 0 as in the commit I mentionned you  https://github.com/odoo/odoo/commit/eb3ed89f8440050fc32f9a7b29fb2b6ab8fd6b59 so visually only the cursor is spinning (light effect vs spin + applying an opacity on top of the whole screen)).

Reply:
I added the opacity to make it visually more clear that the editor is loading. Either way way is fine by me though, I don't feel I'm the right person to argue about this :) Consistency is probably just as important?